projects/go-lib/styles/_forms.scss

```diff
@@ -96,6 +95,12 @@ $input--disabled-background: rgba($base-dark-secondary, .4);
     }
   }
 
+  // TextArea
+  // ---------------------------------------------
+  &__textarea {
+    min-height: 70px;
```

Comment:
This style is for `min-height` but the conversation/testing instructions say `min-width` of 70px. Just want to make sure this is the correct change...

Reply:
@adhiyan-tangoe I am not questioning the requirements of this ticket. The description above says "width":

<img width="482" alt="Screen Shot 2023-03-31 at 7 41 34 AM" src="https://user-images.githubusercontent.com/8953503/229152245-bca445d4-0c90-494b-8753-c82ad3125066.png">

Just making sure the description is wrong, not the actual style...

Reply:
Its supposed to be height not width just to clarify from UX. The image in the initial ticket shows the height issue.

Reply:
@stolve thank you! Approving now.